pkg/resources/configmaps.go

```diff
@@ -46,6 +50,9 @@ const QRadarConfigName = "remote-syslog-config"
 // SplunkConfigName defines the name of the splunk-hec-config configmap
 const SplunkConfigName = "splunk-hec-config"
 
+// HTTPIngestConfigName definest the name of the http-ingesturl-config configmap
+const HTTPIngestConfigName = "http-ingesturl-config"
```

Comment:
I'm not great with naming conventions, but wondering if we should maybe remove the `-config` postfix since it's not configurable?

Reply:
sure